### PR TITLE
Update auto-failover-group-configure-sql-db.md

### PR DESCRIPTION
**Pull request by gabriellysaraiva**
Adding a note just remind that the failover group needs to be created in a different region.

### CODE REVIEW

**Comment by Court72** [comment]
@AbdullahMSFT 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/database/auto-failover-group-configure-sql-db.md [hunk 1]
```diff
@@ -54,6 +54,11 @@ Create your failover group and add your single database to it using the Azure po
 
 1. On the **Failover Group** page, enter or select the required values, and then select **Create**.
 
+
+> [!NOTE]
+> The Failover Group needs to be created in a different region.
```
Comment (rwestMSFT) [inline azure-sql/database/auto-failover-group-configure-sql-db.md]:
```suggestion
> The failover group must be created in a different region.
```

File: azure-sql/database/auto-failover-group-configure-sql-db.md [hunk 2]
```diff
@@ -54,6 +54,11 @@ Create your failover group and add your single database to it using the Azure po
...
+>
```
Comment (rwestMSFT) [inline azure-sql/database/auto-failover-group-configure-sql-db.md]:
```suggestion
```

**Comment by rajeshsetlem** [comment]
Hi @gabriellysaraiva, Thanks for the feedback. In the portal we do have checks to ensure the secondary server in failover group must be in different region,

image

File: azure-sql/database/auto-failover-group-configure-sql-db.md [hunk 3]
```diff
@@ -54,6 +54,10 @@ Create your failover group and add your single database to it using the Azure po
 
 1. On the **Failover Group** page, enter or select the required values, and then select **Create**.
```
Comment (MashaMSFT) [inline azure-sql/database/auto-failover-group-configure-sql-db.md]:
```suggestion
1. On the **Failover Group** page, enter or select the required values, and then select **Create**. The failover group must be created in a different region.
```

File: azure-sql/database/auto-failover-group-configure-sql-db.md [hunk 4]
```diff
@@ -54,6 +54,10 @@ Create your failover group and add your single database to it using the Azure po
...
 
+
+> [!NOTE]
```
Comment (MashaMSFT) [inline azure-sql/database/auto-failover-group-configure-sql-db.md]:
```suggestion
```

File: azure-sql/database/auto-failover-group-configure-sql-db.md [hunk 5]
```diff
@@ -54,6 +54,10 @@ Create your failover group and add your single database to it using the Azure po
...
+> The failover group must be created in a different region.
```
Comment (MashaMSFT) [inline azure-sql/database/auto-failover-group-configure-sql-db.md]:
```suggestion
```

**Review by MashaMSFT — COMMENTED** [review]
removing the note and adding the text to the step instead

**Comment by MashaMSFT** [comment]
@gabriellysaraiva , thanks for your submission and helping improve our documentation! When writing in a sequence of numbers, it's better to include text in-line, vs breaking up the numbers with a note :) I've modified the PR slightly to align with this guidance. Thanks again! #sign-off

**Comment by MashaMSFT** [comment]
@Court72 fyi #sign-off

**Comment by Court72** [comment]
Thanks, @MashaMSFT 

PRMerger will accept sign-off from the listed author only. However, if you are a Microsoft employee authorized to update this content, you can send a request to [techdocprs@microsoft.com](mailto:techdocprs@microsoft.com) for this PR to be merged. Include your role and an explanation of why the PR should be merged without approval from the listed author.

**Comment by MashaMSFT** [comment]
lol @Court72 are you serious

**Pull request merged**